packages/react/src/analytics/integrations/Vitorino/Vitorino.ts

```diff
@@ -72,204 +67,87 @@ export default class Vitorino extends integrations.Integration<VitorinoIntegrati
    * @param analytics - Analytics stripped down instance.
    */
   constructor(
-    options: VitorinoIntegrationOptions,
+    options: VitorinoIntegrationOptions = {},
     loadData: LoadIntegrationEventData,
     analytics: StrippedDownAnalytics,
   ) {
+    const baseOptions: VitorinoIntegrationBaseOptions = omit(options, [
+      'riskified',
+      'forter',
+    ]);
     super(options, loadData, analytics);
 
-    this.isScriptLoaded = false;
+    this.baseOptions = baseOptions;
+    this.options = options;
```

Comment:
This is not necessary since the super will take care of that.

Reply:
Fixed